### PR TITLE
suppress warning

### PR DESCRIPTION
**Pull request by afabri**

### CODE REVIEW

**Comment by sloriot** [comment]
Merged in [CGAL-4.8-Ic-13](https://cgal.geometryfactory.com/CGAL/Members/testsuite/results-4.8-Ic-13.shtml)

**Comment by sloriot** [comment]
Tests look good

File: Mesh_3/demo/Mesh_3/Volume_planes_plugin.cpp [hunk 1]
```diff
@@ -1,5 +1,10 @@
 #include "config.h"
 
+#if defined(BOOST_MSVC)
+#  pragma warning(push)
```
Comment (lrineau) [inline Mesh_3/demo/Mesh_3/Volume_planes_plugin.cpp]: `BOOST_MSCV` cannot be defined if `<boost/config.hpp>` is not included first.

**Comment by lrineau** [comment]
[The warning is still there.][1] The use pf `BOOST_MSVC` requires the inclusion of `<boost/config.hpp>` first.

[1]: https://cgal.geometryfactory.com/CGAL/Members/testsuite/CGAL-4.8-Ic-13/Mesh_3_Demo/TestReport_afabri_x64_Cygwin-Windows8_MSVC2013-Release-64bits.gz

**Comment by sloriot** [comment]
The warning is [fixed](https://cgal.geometryfactory.com/CGAL/Members/testsuite/CGAL-4.8-Ic-15/Mesh_3_Demo/TestReport_afabri_x64_Cygwin-Windows8_MSVC2013-Release-64bits.gz). Merged in master (7a6a846c)

**Pull request merged**